addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar_tests.js

```diff
@@ -394,6 +394,73 @@ QUnit.test('attachment counter with attachments', async function (assert) {
     );
 });
 
+QUnit.test('composer state conserved when clicking on another topbar button', async function (assert) {
+    assert.expect(8);
+
+    await this.start({
+        async mockRPC(route) {
+            if (route.includes('ir.attachment/search_read')) {
+                return [];
+            }
```

Comment:
can't rely on `mock_server:_mockSearchReadController`?

Reply:
yes indeed, I adapt other tests of the file